ravenframework/Optimizers/Optimizer.py

```diff
@@ -148,6 +148,18 @@ def getInputSpecification(cls):
               of the variables. For example, if the Sampler selected samples only 2 of the 5 optimization
               variables, the \xmlNode{initial} XML node is required only for the remaining 3 variables."""))
 
+    model = InputData.assemblyInputFactory('Model', contentType=InputTypes.StringType, strictMode=True,
+        printPriority=175,
+        descr=r"""Name of a Model that optimizers may want to use during optimization. For example, the
+              Bayesian Optimizer requires a ROM to select points during optimization. The model is defined in
+              detail with in the \xmlNode{Models} as in other uses. This node should be provided a string referencing
+              the model definition's name.""")
+    model.addParam('subType', InputTypes.StringType, True,
+        descr=r"""RAVEN subType for this source. For example, \xmlNode{GaussianProcessRegressor} for the
+              GPR model within RAVEN. For more information on specifying ROM's in RAVEN, see the relevant
+              section in the user manual.""")
+
+    specs.addSub(model)
```

Comment:
I suggest to use 'ROM' node, see my previous comments. In addition, I think we may need to generalized it in Sampler class, right now, 'ROM' is used in Sobol, SparseGrid, LimitSurface, etc.